src/composables/useFlaws.ts

```diff
@@ -0,0 +1,64 @@
+import {  ref } from 'vue';
+import { getFlaws } from '@/services/FlawService';
+
+export function useFlaws() {
+  const isFinalPageFetched = ref(false);
+  const isLoading = ref(false);
+  const issues = ref<any[]>([]);
+  const offset = ref(0);
+  const pagesize = 20;
+
+  function loadFlaws(params: any = {}) {
+    offset.value = 0;
+    isFinalPageFetched.value = false;  
+    isLoading.value = true;
+    issues.value = [];
+
+    getFlaws(offset.value, 100, params.value)
+      .then((response) => {
+        if (response.data.results.length < pagesize) {
+          isFinalPageFetched.value = true;
+        }
+        issues.value = response.data.results;
+        offset.value += response.data.results.length; // Increase the offset for next fetch
```

Comment:
`response.data` has some properties that handle pagination offset values as well, I forget, can we use that to determine the offset? I guess there may not be a functional difference, if that's the case this is fine. I seem to recall this working well.

Reply:
response.data has only `count` property for all issue count.

Reply:
There should be two additional properties, `previous` and `next` which basically includes crafted URLS with offset already applied based on limit. When `next` is empty, you are on the last page.

Reply:
Thanks for your information. @JakubFrejlach  I will use next property for offset.